Embeds the DevTools extensions video (#10427)
flutter/website: src/content/tools/devtools/extensions.md

## What are DevTools extensions?

[DevTools extensions][]
are developer tools provided by third-party packages that are
tightly integrated into the DevTools tooling suite.
Extensions are distributed as part of a pub package,
and they are dynamically loaded into DevTools when
a user is debugging their app.

[DevTools extensions]: {{site.pub-pkg}}/devtools_extensions

## Use a DevTools extension

> If your app depends on a package that provides a
> DevTools extension, the extension automatically
> shows up in a new tab when you open DevTools.

Reviewer: **nitpick:** Remove "automatically". If a user has to do something, it's manual. If a program or daemon does it, it's automatic. The adverb would be redundant. Also, condition before action, per GSG.
```suggestion
When you open DevTools and your app depends on
a package that provides a DevTools extension,
the extension displays in a new tab.
```

### Configure extension enablement states

> You need to manually enable the extension before it loads

Reviewer: **nitpick:** Remove "manually". If a user has to do something, it's manual. If a program or daemon does it, it's automatic. The adverb would be redundant.

```suggestion
Before the extension loads, you need to enable it
```

> This file stores per-project
> (or optionally, per user) settings for DevTools.

Reviewer: **nitpick:** This looks to be more of a default vs. option rather than noting it as an optional setting.

```suggestion
This file stores DevTools settings either per project (default)
or per user.
How the file stores settings depends upon whether
the file is checked into source control.
```

> If this file is **checked into source control**,
> the specified options are configured for the project.
> This means that anyone who pulls a project's
> source code and works on the project uses the same settings.

Reviewer: **todo:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
If you **check this file into source control**,
DevTools configures the specified options for the project.
This means that anyone who pulls a project's
source code and works on the project uses the same settings.
```

> If this file is **omitted from source control**,
> for example by adding `devtools_options.yaml`
> as an entry in the `.gitignore` file, then the specified
> options are configured separately for each user.

Reviewer: **todo:** 
- Write in active voice per [GSG](https://developers.google.com/style/voice). 
- Write as shorter sentences per [GSG](https://developers.google.com/style/accessibility#ease-of-reading).

```suggestion
If you **omit this file from source control**,
DevTools configures the specified options per user.
For example, if you add `devtools_options.yaml`
as an entry in the `.gitignore` file,
`git` omits the file from source control.
```
